docs/installation.rst

```diff
@@ -142,6 +142,7 @@ following dependencies first:
 * OpenSSL_ >= 1.0.0, plus development headers.
 * libacl_ (which depends on libattr_), both plus development headers.
 * liblz4_, plus development headers.
+* ZeroMQ_ >= 4.0.0, plus development headers.
```

Comment:
i noticed that binary builds of the pyzmq package contain libzmq, so in that case, it is not needed to install them separately (at least as we do not need it for our own code).